engine/src/main/scala/cromwell/engine/workflow/lifecycle/MaterializeWorkflowDescriptorActor.scala

```diff
@@ -429,6 +430,16 @@ class MaterializeWorkflowDescriptorActor(serviceRegistryActor: ActorRef,
     }
   }
 
+  private def validateWdlFiles(workflowInputs: WorkflowCoercedInputs): ErrorOr[WorkflowCoercedInputs] = {
+    val failedFiles = workflowInputs.collect {
+      case (fullyQualifiedName , WdlSingleFile(value)) if value.startsWith("\"gs://") => "Invalid value for File input '" + fullyQualifiedName  + "':"+ value + " starts with a '\"' "
```

Comment:
Ah thanks for the extra prod.  I was getting compilation errors and then I saw this - https://github.com/scala/bug/issues/6476 so I had all but given up but `"""` interpolation works which i just found out was a thing!